### ISSUE TITLE
[at-loader]: Child process failed to process the request:  TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by cbll** [s-panferov/awesome-typescript-loader#505]
I have two very weird cases I'd love to get some help with!

Case 1: 
`"awesome-typescript-loader": "3.1.0",`

On OSX, it will throw this error:

```
[at-loader] Checking finished with 1 errors
[at-loader]: Child process failed to process the request:  TypeError: Cannot read property '0' of undefined
```

Then we think: OK, let's upgrade to:
`"awesome-typescript-loader": "3.1.3",`

And it works on OSX! Except, we build on Jenkins(Ubuntu)... And while Jenkins is happy about the 3.1.0 version and build with no issues, upgrading to 3.1.3 throws the following:

```
[at-loader] Checking finished with 2 errors
Error in bail mode: [at-loader] ./src/pages/somePage.tsx:4:23
    TS2307: Cannot find module 'History'.
```

.. This is related to React and React-router, I think? But why this module specifically?

In other words, we're in a position where we either cannot build without errors locally, or cannot build on a build server without errors. 

Any ideas? I'm pulling head out of my hair at this stage :D
